vscode-extensions/tls-sync-vscode-ui-extension/.gitignore

```diff
@@ -0,0 +1,2 @@
+.vscode-test/
+*.vsix
```

Comment:
The .vsix file should be produced in an existing, known output folder (ex. `/dist`). Why are we explicitly ignoring it here?

Reply:
I'll remove .vsix filter and move `.vscode-test` to the root `.gitignore`. I didn't find a config to update this. `.vscode-test` contains temp files including a copy of vscode used for testing.